src/ui/public/styles/sidebar.less

```diff
@@ -123,19 +135,27 @@
   }
 
   .index-pattern {
-    background-color: @sidebar-active-bg;
     font-weight: bold;
     padding: 5px 10px;
-    color: @sidebar-active-color;
     display: flex;
     justify-content: space-between;
+    background-color: @kibanaGray4;
+    line-height: 28px;
```

Comment:
Specifically line height though, I assume this is correlated to the icon size somehow?